### PR TITLE
Breadcrumb Component

### LINKED ISSUE #3: Breadcrumb component
**Description of feature/enhancement**
Create the common breadcrumb component. content not yet finalised but this will be the layout. Each of the values are also clickable and will bring you to the correct page

image_2022-10-18_20-31-57

### PR DESCRIPTION
**Pull request by Enigmatrix**
Closes #3.

**Overview of changes:**

Add breadcrumb component.

**Anything you'd like to highlight/discuss:**

Component usage is based on the [Tabs example](https://mui.com/material-ui/react-tabs/#basic-tabs), but replace `Tabs` with `Breadcrumbs` and `Tab` with `Breadcrumb`.

### CODE REVIEW

File: components/Breadcrumbs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ *       <div hidden={value != 0}>Item One</div>
+ *       <div hidden={value != 1}>Item Two</div>
+ *       <div hidden={value != 2}>Item Three</div>
+ *       <div hidden={value != 3}>Item Four</div>
+ *       <div hidden={value != 4}>Item Five</div>
+ *     </div>
+ *   )
+ * }
+ * ```
+ */
```
Comment (wilinetan) [inline components/Breadcrumbs/index.tsx]: is this like a special documentation function?

File: pages/index.tsx [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+          <Breadcrumb label="Donation Amount" />
+          <Breadcrumb label="My Details" />
+          <Breadcrumb label="Submit" />
+        </Breadcrumbs>
+      </Box>
+      <div hidden={value != 0}>Item One</div>
+      <div hidden={value != 1}>Item Two</div>
+      <div hidden={value != 2}>Item Three</div>
+      <div hidden={value != 3}>Item Four</div>
+      <div hidden={value != 4}>Item Five</div>
```
Comment (wilinetan) [inline pages/index.tsx]: use !== instead of !=

File: components/Breadcrumbs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import Tabs, { TabsProps } from '@mui/material/Tabs'
+
+type BreadcrumbsProps = TabsProps
```
Comment (wilinetan) [inline components/Breadcrumbs/index.tsx]: can do away with this its just referencing TabsProps. use TabsProps directly in ur component

**Review by wilinetan — COMMENTED** [review]
good work! 😄

File: components/Breadcrumbs/index.tsx (hunk 1 above)
Comment (Enigmatrix) [reply components/Breadcrumbs/index.tsx]: yep! I'll add it in to notion docs soon

image

**Review by wilinetan — APPROVED** [review]
LGTM. can merge after u remove the breadcrumb page. Thanks! 👍

**Pull request merged**